starwars/src/App.js

```diff
@@ -8,6 +8,8 @@ class App extends Component {
     super();
```

Comment:
- Great job on the sprint challenge Timi. 
- You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.

- You have also gone above and beyond and completed a stretch goal to build pagination system. You are able to load and display next or previous set of data on Your app without refreshing the page based on React convention.
Your code is clean and Your git workflow is fluid.
Overall:
* Able to fetch and display data (MVP)
* Good answers
* Possible to switch to next “page” by loading next set of data (stretch)
* Clean and modular code with multiple components
* Fluid git workflow